src/components/MAPI/organizations/AccessControl/AccessControlRoleSubjects.tsx

```diff
@@ -146,6 +146,35 @@ const mapValueToSetItem = (stateValue: IStateValue) => (
   };
 };
 
+const getBindServiceAccountSuccessMessage = (
+  values: string[],
+  statuses: ui.AccessControlRoleSubjectStatus[],
+  statusFilter: ui.AccessControlRoleSubjectStatus
+): string => {
+  const accounts = values.filter((_, i) => statuses[i] === statusFilter);
```

Comment:
We do not enforce in any way that for a value in `values` we also have a status in `statuses`, and that it is present at the same index. This isn't very safe, as the implementation of this depends on `handleAdd()` in `src/components/MAPI/organizations/AccessControl/index.tsx`.

Do you think it would be worth combining the value and status into a single data structure?